receiver/otelarrowreceiver/testdata/config.yaml

```diff
@@ -27,5 +27,6 @@ protocols:
         permit_without_stream: true
   arrow:
     memory_limit_mib: 123
+  admission:
```

Comment:
Since these tests are easy to write and add more of, please add another `.yaml` file that uses the legacy names inside the `arrow` config block, to ensure they can still be parsed and validated as we expect.

Reply:
Added a new test to confirm validation of deprecated fields will set the new `admission` config block